src/Service/PageTitle.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Service;
+
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class PageTitle
+{
+    public function __construct(
+        private readonly BreadcrumbTrail $breadcrumbTrail,
+        private readonly Fallbacks $fallbacks,
+        private readonly TranslatorInterface $translator,
+    ) {
+    }
+
+    public function getTitle(): string
```

Comment:
How can I overrule the pagetitle? By setting it in the template and overruling `{%block title %}`? 
As there is a service, wouldn't it make more sense to have some method to set the title? 

In my opinion the reversed breadcrumb is the fallback if no specific title is set. But I think it should work in the same way the breadcrumbs work. See https://github.com/sumocoders/FrameworkCoreBundle/blob/master/docs/development/breadcrumb.md. But with `#[Pagetitle(XXX)]`? But maybe something we should discuss with other devs.

Reply:
It would be done by setting the {% block title %}, an attribute would work but seems a bit overkill.
@jonasdekeukelaere @davysumo @absumo  What's your opinion

Reply:
`{%block title %}` for now might be enough? Unless you think the reversed breadcrumb is not enough for a default? By looks of it, we didn't even set custom titles before?
PageTitle attribute will need some rework as you'll need a listener to set the page title based on the attribute, probably with an option to append or overwrite? Plus this PageTitle class probably will need a setter for the setting the title by the listener?

Reply:
>  By looks of it, we didn't even set custom titles before?
🙄